src/test/fixtures/profiles/processed-profile.js

```diff
@@ -91,32 +91,6 @@ export type TestDefinedJsTracerEvent = [
   Milliseconds
 ];
 
-/**
- * This function ensures that the mock payloads are converted correctly to real payloads
- * that match the MarkerPayload typing. Specifically it adds the 'DummyForTests' type
- * to { startTime, endTime } payloads. Doing this means that it's cleaner to create
- * dummy test-defined markers, since we don't have to add a `type` to the payload.
- */
-function _refineMockPayload(
-  payload: MarkerPayload | MockPayload
-): MarkerPayload {
-  if (
-    // Check for a MockPayload.
-    payload !== null &&
-    Object.keys(payload).length === 2 &&
-    typeof payload.startTime === 'number' &&
-    typeof payload.endTime === 'number'
-  ) {
-    return {
-      type: 'DummyForTests',
-    };
-  }
-  // There is no way to refine the payload type to just the { startTime, endTime }
-  // mock marker. So check for those conditions above, and coerce the final result
-  // into a MarkerPayload using the function signature.
-  return (payload: any);
-}
-
```

Comment:
I realize that this isn't needed anymore. In the past we used to have a payload like
```
{
  type: 'DummyForTests',
  startTime: 111,
  endTime: 222,
}
```
But this was changed when start and end times both moved to the main tuple. As a result this refinement is unnecessary now.

Reply:
Thanks for the cleanup!